Trend Micro/email-security/tests/test_quarantined.json

```
@@ -0,0 +1,41 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Trend Micro Email Security",
+        "dialect_uuid": "2e9d87ed-6606-445a-90d1-9c7695b28335"
+      }
+    },
+    "message": "{\"size\": 51149, \"action\": \"Quarantined\", \"mailID\": \"b879ff84-55a3-4813-be99-9e0386a446f7\", \"sender\": \"john.doe@example.org\", \"genTime\": \"2023-09-28T13:47:18Z\", \"subject\": \"My beautiful subject\", \"headerTo\": [\"jane.doe@example.org\"], \"direction\": \"in\", \"messageID\": \"<11111111111111111111111111111111111111111111111111111111111111111@example.org>\", \"recipient\": \"jane.doe@example.org\", \"timestamp\": \"2023-09-28T13:45:59Z\", \"headerFrom\": \"john.doe@example.org\", \"embeddedUrls\": [\"https://sekoia.io\", \"https://www.nytimes.com\"]}"
+  },
+  "expected": {
+    "message": "{\"size\": 51149, \"action\": \"Quarantined\", \"mailID\": \"b879ff84-55a3-4813-be99-9e0386a446f7\", \"sender\": \"john.doe@example.org\", \"genTime\": \"2023-09-28T13:47:18Z\", \"subject\": \"My beautiful subject\", \"headerTo\": [\"jane.doe@example.org\"], \"direction\": \"in\", \"messageID\": \"<11111111111111111111111111111111111111111111111111111111111111111@example.org>\", \"recipient\": \"jane.doe@example.org\", \"timestamp\": \"2023-09-28T13:45:59Z\", \"headerFrom\": \"john.doe@example.org\", \"embeddedUrls\": [\"https://sekoia.io\", \"https://www.nytimes.com\"]}",
+    "event": {
+      "kind": "event",
+      "category": ["email"],
+      "type": ["info"],
+      "action": "Quarantined"
+    },
+    "@timestamp": "2023-09-28T13:45:59Z",
+    "email": {
+      "local_id": "b879ff84-55a3-4813-be99-9e0386a446f7",
+      "sender": {
+        "address": "john.doe@example.org"
+      },
+      "subject": "My beautiful subject",
+      "from": {
+        "address": "john.doe@example.org"
+      },
+      "to": {
+        "address": ["jane.doe@example.org"]
+      },
+      "message_id": "<11111111111111111111111111111111111111111111111111111111111111111@example.org>",
```

I suggest removing '<' and '>' from the message_id.
```suggestion
      "message_id": "11111111111111111111111111111111111111111111111111111111111111111@example.org",
```